aws/scripts/py/deploy_content.py

```
@@ -19,7 +17,7 @@
     print("Fetching CloudFront distributions...")
     result = subprocess.check_output(
         ['aws', 'cloudfront', 'list-distributions',
-         '--region', CLOUDFRONT_REGION, '--no-cli-pager',]
+         '--region', CLOUDFRONT_REGION, '--no-cli-pager']
```

Ruff

Starting a process with a partial executable path

(S607)